server/channels/app/login_test.go

```diff
@@ -48,22 +48,31 @@ func TestCWSLogin(t *testing.T) {
 
 	t.Run("Should authenticate user when CWS login is enabled and tokens are equal", func(t *testing.T) {
 		token := model.NewToken(TokenTypeCWSAccess, "")
-		defer th.App.DeleteToken(token)
+		defer func() {
+			appErr := th.App.DeleteToken(token)
+			require.Nil(t, appErr)
+		}()
+
 		os.Setenv("CWS_CLOUD_TOKEN", token.Token)
-		user, err := th.App.AuthenticateUserForLogin(th.Context, "", th.BasicUser.Username, "", "", token.Token, false)
-		require.Nil(t, err)
+		user, appErr := th.App.AuthenticateUserForLogin(th.Context, "", th.BasicUser.Username, "", "", token.Token, false)
+		require.Nil(t, appErr)
 		require.NotNil(t, user)
 		require.Equal(t, th.BasicUser.Username, user.Username)
-		_, apperr := th.App.Srv().Store().Token().GetByToken(token.Token)
-		require.NoError(t, apperr)
-		th.App.DeleteToken(token)
+		_, err := th.App.Srv().Store().Token().GetByToken(token.Token)
+		require.NoError(t, err)
+		appErr = th.App.DeleteToken(token)
+		require.NoError(t, appErr)
```

Comment:
This should be require.Nil

Reply:
Should it still be require.NoError for the above err, but update just the lower appErr to require,Nil, or update both?

Sorry for needing so many updates on this PR 😅

Reply:
You only need to update this line to use `require.Nil(t, appErr)`

No problem. Keep asking questions!